### DISCUSSION TITLE
Using LineSegmentIntersector with osgEarthTriton

### DISCUSSION THREAD

**Discussion by kshahar** [pelicanmapping/osgearth#1956] [Support Q&A]
Perhaps this is more of an OSG question than osgEarth, but how would you suggest to integrate `osgEarth/DPLineSegmentIntersector` with osgEarthTriton? I am trying to think of a clean way to call Triton's GetHeight while the intersection test is performed.

**Comment by gwaldron** [comment]
Yeah, there is nothing in osgEarth to help with that. Suggest looking at the Triton API and see if there are any options there.
